ci/pyuvdata_min_deps_tests.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - conda-forge
 dependencies:
   - astropy
-  - nose
   - numpy
   - scipy
   - six
   - coverage
+  - pytest-cov
```

Comment:
To my knowledge the `pytest-cov` is a plugin which uses `coverage` and formats the outputs differently. We don't need to specifically declare `coverage` since conda will grab it anyway. It can be removed if you want.

Reply:
ok, I'm fine with leaving it if it doesn't add something we don't need.